src/ReactiveUI/ReactiveObject/IReactiveObjectExtensions.cs

```diff
@@ -48,12 +48,22 @@ private interface IExtensionState<out TSender>
             /// </summary>
             IObservable<Exception> ThrownExceptions { get; }
 
+            /// <summary>
+            /// sdfsdg.
```

Comment:
We probably want a real comment here

Reply:
I absolutely agree. That was an oversight on my part. Sorry about that. I'll fix this one and the other one below.